ipasetup.py.in

```diff
@@ -30,14 +30,21 @@ common_args = dict(
     platforms=["Linux", "Solaris", "Unix"],
     classifiers=[
         "Development Status :: 5 - Production/Stable",
-        ("Topic :: System :: Systems Administration :: "
-         "Authentication/Directory :: LDAP"),
-        "Topic :: Internet :: Name Service (DNS)",
-        "Intended Audience :: System Environment/Base",
-        "License :: GPL",
-        "Programming Language :: Python",
+        "Intended Audience :: System Administrators",
+        ("License :: OSI Approved :: "
+         "GNU General Public License v3 or later (GPLv3+)"),
+        "Programming Language :: C",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: Implementation :: CPython",
         "Operating System :: POSIX",
+        "Operating System :: POSIX :: Linux",
         "Operating System :: Unix",
+        "Topic :: Internet :: Name Service (DNS)",
+        ("Topic :: System :: Systems Administration :: "
+         "Authentication/Directory :: LDAP"),
```

Comment:
None of the PyPI packages will contain a WSGI application. The WSGI is in ipaserver but we are not going to release the server on PyPI yet.